exercises/collatz-conjecture/collatz_conjecture.exs

```diff
@@ -0,0 +1,13 @@
+defmodule CollatzConjecture do
+
+  @doc """
+  calc/1 takes an integer and returns the number of steps required to get the
+  number to 1 when following the rules:
+    - if number is odd, multiply with 3 and add 1
+    - if number is even, divide by 2
+  """
+  @spec calc(number :: integer) :: pos_integer
+  def calc(input) do
+    raise "ain't got no time for that"
```

Comment:
In general we don't start with an error when we give the function definition. I think you can just cut this line.